framework/src/meshgenerators/AdvancedExtruderGenerator.C

```diff
@@ -1024,12 +1024,14 @@ AdvancedExtruderGenerator::generate()
     }
   }
 
+#ifdef LIBMESH_ENABLE_UNIQUE_ID
```

Comment:
I approve as a matter of principle, but I don't think MOOSE has compiled without unique_id support for years.